shuffle/integration-tests/src/helper.rs

```diff
@@ -2,46 +2,67 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use anyhow::Result;
-use diem_sdk::{client::BlockingClient, types::LocalAccount};
-use shuffle::{account, deploy, new, shared::Home};
+use diem_sdk::{
+    client::BlockingClient, transaction_builder::TransactionFactory, types::LocalAccount,
+};
+use shuffle::{
+    account, deploy, new,
+    shared::{Home, NetworkHome},
+};
 use std::{path::PathBuf, str::FromStr};
 use tempfile::TempDir;
 use url::Url;
 
 pub struct ShuffleTestHelper {
     home: Home,
+    network_home: NetworkHome,
     tmp_dir: TempDir,
 }
 
 impl ShuffleTestHelper {
     pub fn new() -> Result<Self> {
         let tmp_dir = TempDir::new()?;
         let home = Home::new(tmp_dir.path())?;
-        Ok(Self { tmp_dir, home })
+        let network_home = NetworkHome::new(tmp_dir.path());
+        Ok(Self {
+            tmp_dir,
+            home,
+            network_home,
+        })
     }
 
     pub fn home(&self) -> &Home {
         &self.home
     }
 
+    pub fn network_home(&self) -> &NetworkHome {
+        &self.network_home
```

Comment:
Can we have this be `&self.home.networks.get("localhost")?;`? How would you create a custom network home w the connectivity information given to use by forge ctx?